backend/tests/v5/e2e/routes/teamspaces/projects/models/common/general.test.js

```diff
@@ -65,10 +67,16 @@ const generateBasicData = () => {
 		calibration: ServiceHelper.generateCalibration(),
 	};
 
-	data.jobs = [
-		{ _id: ServiceHelper.generateRandomString(), users: [viewer.user] },
-		{ _id: ServiceHelper.generateRandomString(), users: [collaborator.user] },
-		{ _id: ServiceHelper.generateRandomString(), users: Object.values(data.users).map(({ user }) => user) },
+	data.roles = [
```

Comment:
since we're doing this in multiple places, can we just have a generateRole that takes in a user array and use that everywhere?